File: 2st/Riyeon/주사위 게임 1.py
def solution(a, b):
    answer = 0
    if a % 2 == 1 and b % 2 == 1:
        return a*a + b*b
    elif a % 2 == 1 or b % 2 == 1:
        return 2 * (a + b)
    return abs(a-b)

File: 4st/Hani/programmer/주사위 게임 1.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        if (a % 2 == 1 && b % 2 == 1) {
            answer = (a * a) + (b * b);
        } else if (a % 2 == 1 || b % 2 == 1) {
            answer = 2 * (a + b);
        } else if (a % 2 == 0 && b % 2 == 0) {
            answer = a - b;
            if (answer < 0) {
                answer = -1 * answer;
            }
        }
        return answer;
    }
}